Sources/swift-test-codecov/main.swift

```diff
@@ -83,6 +83,13 @@ struct StatsCommand: ParsableCommand {
     )
     var includeDependencies: Bool = false
 
+    @Flag(
+        name: [.customLong("tests")],
+        inversion: .prefixedNo,
+        help: ArgumentHelp("Determines whether test files are included in coverage calculation. (default: false)")
```

Comment:
ArgumentParser will add the (default: false) part at the end so you can leave it off your help string here.

Reply:
Oh I see. Got it, removing that part.